pkgs/development/interpreters/python/cpython/3.6/default.nix

```diff
@@ -64,6 +64,14 @@ in stdenv.mkDerivation {
     substituteInPlace configure --replace '-Wl,-stack_size,1000000' ' '
   '';
 
+  patches = optionals (!(stdenv.cc.isGNU or false)) [
```

Comment:
Ok. In that case I should probably boot up a little NixOS VM and try if that this also works fine. BTW: The python27 currently also does this only on !isGNU, should we maybe change that?